client/src/styles/Home.css

```diff
@@ -32,3 +32,12 @@
   text-align: center;
   margin-top: 20px;
 }
+
+.footerStyle:hover {
+  text-decoration: underline;
+  color: white;
+}
+
+a{
+  color:white
```

Comment:
If you do that it will change the color to ALL the link as white, which we don't want to do :) find a different solution.